Antidote/RingingCallViewController.m

```diff
@@ -61,6 +56,42 @@ - (void)installConstraints
         make.width.equalTo(kButtonSize);
         make.centerY.equalTo(self.view.centerY).with.offset(kIndent);
     }];
+
+    [self.incomingCallLabel makeConstraints:^(MASConstraintMaker *make) {
+        make.top.equalTo(self.nameLabel.bottom).with.offset(kIndentBelowNameLabel);
+        make.centerX.equalTo(self.view.centerX);
+    }];
+}
+
+#pragma mark - View setup
+
+- (void)createAcceptCallButton
+{
+    self.acceptCallButton = [UIButton new];
+    self.acceptCallButton.backgroundColor = [UIColor greenColor];
+    [self.acceptCallButton setImage:[UIImage imageNamed:@"call-accept"] forState:UIControlStateNormal];
+    [self.acceptCallButton addTarget:self action:@selector(acceptCallButtonPressed) forControlEvents:UIControlEventTouchUpInside];
+
+    [self.view addSubview:self.acceptCallButton];
+}
+
+- (void)createDeclineCallButton
+{
+    self.declineCallButton = [UIButton new];
+    self.declineCallButton.backgroundColor = [UIColor redColor];
+    [self.declineCallButton setImage:[UIImage imageNamed:@"call-decline"] forState:UIControlStateNormal];
+    [self.declineCallButton addTarget:self action:@selector(endCall) forControlEvents:UIControlEventTouchUpInside];
+
+    [self.view addSubview:self.declineCallButton];
+}
+
+- (void)createIncomingCallLabel
+{
+    self.incomingCallLabel = [UILabel new];
+    self.incomingCallLabel.text = @"incoming call";
```

Comment:
Please use `NSLocalizedString` for all strings, with component name as a second parameter ("Calls" in you case).